apps/builder/app/routes/s/css.ts

```diff
@@ -47,7 +46,7 @@ export const loader = async ({ request }: ActionArgs) => {
         assets: canvasData.assets,
         breakpoints: canvasData.build?.breakpoints,
         styles: canvasData.build?.styles,
-        styleSourceSelections: canvasData.tree?.styleSourceSelections,
+        styleSourceSelections: canvasData.build?.styleSourceSelections,
```

Comment:
please add to saas link to this change, we have the same code there.

Reply:
Created an issue